<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="toolBox">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_list_widget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>140</width>
          <height>375</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QListWidget 操作</string>
        </attribute>
        <widget class="QToolButton" name="tbtnListIni">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>85</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>tbtnListIni</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
        <widget class="QToolButton" name="tbtnListClear">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>97</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>tbtnListClear</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
        <widget class="QToolButton" name="tbtnListInsert">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>103</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>tbtnListInsert</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
        <widget class="QToolButton" name="tbtnListAppend">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>103</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>tbtnListAppend</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
        <widget class="QToolButton" name="tbtnListDelete">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>103</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>tbtnListDelete</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_tree_widget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>140</width>
          <height>375</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QTreeWidget 操作</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_table_widget">
        <attribute name="label">
         <string>QTableWidget 操作</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabQListWidget">
        <attribute name="title">
         <string>tabQListWidget</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>当前项变化</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editCutItemText">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>600</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 255);</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chkBoxListEditable">
                <property name="text">
                 <string>可编辑</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QToolButton" name="tBtnSelectItem">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>tBtnSelectItem</string>
                </property>
                <property name="popupMode">
                 <enum>QToolButton::MenuButtonPopup</enum>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="tBtnSelALL">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>tBtnSelALL</string>
                </property>
                <property name="popupMode">
                 <enum>QToolButton::DelayedPopup</enum>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="tBtnSelNone">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>tBtnSelNone</string>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="tBtnSelInvs">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>tBtnSelInvs</string>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QListWidget" name="listWidget">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <item>
            <property name="text">
             <string>item 1</string>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
            <property name="icon">
             <iconset resource="../req.qrc">
              <normaloff>:/images/image/check2.ico</normaloff>:/images/image/check2.ico</iconset>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item>
            <property name="text">
             <string>item 2</string>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
            <property name="icon">
             <iconset resource="../req.qrc">
              <normaloff>:/images/image/check2.ico</normaloff>:/images/image/check2.ico</iconset>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabQTreeWidget">
        <attribute name="title">
         <string>tabQTreeWidget</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabQTableWidget">
        <attribute name="title">
         <string>tabQTableWidget</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actListIni"/>
   <addaction name="actListClear"/>
   <addaction name="actListInsert"/>
   <addaction name="actListAppend"/>
   <addaction name="actListDelete"/>
   <addaction name="separator"/>
  </widget>
  <action name="actListIni">
   <property name="icon">
    <iconset resource="../req.qrc">
     <normaloff>:/images/image/128.bmp</normaloff>:/images/image/128.bmp</iconset>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="toolTip">
    <string>初始化列表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actListClear">
   <property name="icon">
    <iconset resource="../req.qrc">
     <normaloff>:/images/image/delete1.bmp</normaloff>:/images/image/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="toolTip">
    <string>清除列表</string>
   </property>
  </action>
  <action name="actListInsert">
   <property name="icon">
    <iconset resource="../req.qrc">
     <normaloff>:/images/image/424.bmp</normaloff>:/images/image/424.bmp</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actListAppend">
   <property name="icon">
    <iconset resource="../req.qrc">
     <normaloff>:/images/image/316.bmp</normaloff>:/images/image/316.bmp</iconset>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>添加项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actListDelete">
   <property name="icon">
    <iconset resource="../req.qrc">
     <normaloff>:/images/image/324.bmp</normaloff>:/images/image/324.bmp</iconset>
   </property>
   <property name="text">
    <string>删除当前项</string>
   </property>
   <property name="toolTip">
    <string>删除当前项</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actSelALL">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
  </action>
  <action name="actSelNone">
   <property name="text">
    <string>全不选</string>
   </property>
   <property name="toolTip">
    <string>全不选</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actSelInvs">
   <property name="text">
    <string>反选</string>
   </property>
   <property name="toolTip">
    <string>反选</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="../req.qrc">
     <normaloff>:/images/image/exit.bmp</normaloff>:/images/image/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actSelPopMenu">
   <property name="icon">
    <iconset resource="../req.qrc">
     <normaloff>:/images/image/124.bmp</normaloff>:/images/image/124.bmp</iconset>
   </property>
   <property name="text">
    <string>项选择</string>
   </property>
   <property name="toolTip">
    <string>项选择</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../req.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
